WalletWasabi.Fluent/ViewModels/Wallets/Send/AdvancedSendOptionsViewModel.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.Globalization;
+using System.Reactive.Linq;
+using NBitcoin;
+using ReactiveUI;
+using WalletWasabi.Fluent.Validation;
+using WalletWasabi.Fluent.ViewModels.Navigation;
+using WalletWasabi.Models;
+
+namespace WalletWasabi.Fluent.ViewModels.Wallets.Send
+{
+	[NavigationMetaData(Title = "Advanced")]
+	public partial class AdvancedSendOptionsViewModel : RoutableViewModel
+	{
+		private readonly TransactionInfo _transactionInfo;
+
+		[AutoNotify] private string _customFee;
+
+		public AdvancedSendOptionsViewModel(TransactionInfo transactionInfo)
+		{
+			_transactionInfo = transactionInfo;
+
+			_customFee = transactionInfo.CustomFeeRate != FeeRate.Zero
+				? transactionInfo.CustomFeeRate.SatoshiPerByte.ToString(CultureInfo.InvariantCulture)
+				: "";
+
+			EnableBack = false;
+			SetupCancel(enableCancel: true, enableCancelOnEscape: true, enableCancelOnPressed: true);
+
+			this.ValidateProperty(x => x.CustomFee, ValidateCustomFee);
+
+			var nextCommandCanExecute =
+				this.WhenAnyValue(x => x.CustomFee)
+					.Select(_ =>
+					{
+						var noError = !Validations.Any;
+						var somethingFilled = CustomFee is not null or "";
+
+						return noError && somethingFilled;
+					});
+
+			NextCommand = ReactiveCommand.Create(OnNext, nextCommandCanExecute);
+		}
+
+		private void OnNext()
+		{
+			_transactionInfo.CustomFeeRate = decimal.TryParse(CustomFee, out var customFee) ? new FeeRate(customFee) : FeeRate.Zero;
+
+			Navigate().Back();
+		}
+
+		private void ValidateCustomFee(IValidationErrors errors)
+		{
+			var customFeeString = CustomFee;
+
+			if (customFeeString is null or "")
+			{
+				return;
+			}
+
+			if (!decimal.TryParse(customFeeString, out var value))
```

Comment:
Here too. I almost never review anything related to UI so please make sure you are using `en-US` to parse bitcoin amount.

Reply:
done